bench_column_take.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from utils import make_gather_map
+
+
+@pytest.mark.parametrize("nullify", [True, False])
+@pytest.mark.parametrize("gather_how", ["sequence", "reverse", "random"])
+def test_gather_single_column(benchmark, col, gather_how, nullify):
+    gather_map = make_gather_map(int(col.size * 0.4), col.size, gather_how)
```

Comment:
Should this `int` cast go into `make_gather_map` instead? A gather map is integral by definition, so it seems like you'd want to automatically cast (maybe throwing an error or a warning if `int(len_gather_map) != len_gather_map`, in which case you'd still need to add a check at the call site). What do you think?

Reply:
I think we just need to be explicit about the type that arguments of `make_gather_map` expects with type hints. I'm ok either way. Putting the cast inside the helper certainly gives some developer convenience because creating a gather map with length to some fraction of the length of a column is IMO a quite frequent usage. I'm not so sure about if we need to raise an error or just be explict in the docstring about the truncation that would happen. For this function specifically I don't think the choice of round up/round down is particular an important issue here.

Reply:
I agree. This is just a convenience, so I think putting the rounding into the function and just adding some documentation that it's happening is good enough. Looks like you've already done that.